app/src/main/java/com/powerhungers/fomezero/base_app/feature/login/di/loginModule.kt

```diff
@@ -0,0 +1,12 @@
+package com.powerhungers.fomezero.base_app.feature.login.di
+
+import com.powerhungers.fomezero.base_app.feature.login.domain.usecase.LoginUseCase
+import com.powerhungers.fomezero.base_app.feature.login.presentation.LoginViewModel
+import org.koin.androidx.viewmodel.dsl.viewModel
+import org.koin.dsl.module
+
+val loginModule = module {
+
+    viewModel { LoginViewModel(loginUseCase = LoginUseCase(repository = get(), get = get())) }
```

Comment:
```suggestion
    viewModel {
        LoginViewModel(
            loginUseCase = LoginUseCase(
                repository = get(),
                get = get()
            )
        )
    }
```
melhorar indentação pra facilitar a leitura